src/js/View/ProcessView.js

```diff
@@ -0,0 +1,36 @@
+import { NAME, SELECTOR } from '../constants';
+import View from './View';
+
+export default class ProcessView extends View {
+ constructor(target, model) {
+  super(target, model);
+  this.model.subscribe(this.render.bind(this));
+ }
+
+ #getCarTemplate([carName, carPosition]) {
+  return String.raw`<div class="mr-2">
+    <div class="car-player">${carName}</div>
+    ${Array.from(
+     { length: carPosition },
+     () => '<div class="forward-icon mt-2">⬇️️</div>'
+    ).join('')}
+    
+    ${
+     this.model.isFinished()
+      ? ''
+      : `<div class="mt-2 d-flex justify-center">
+    <div class="relative spinner-container">
+    <span class="d-flex material spinner"></span>
+    </div>
+    </div>`
+    }
+    </div>
+`;
+ }
+ getTemplate() {
+  const positions = this.model.getCarsPosition();
+  return String.raw`<div class="mt-4 d-flex">
+    ${positions.map((value) => this.#getCarTemplate(value)).join('')}
```

Comment:
```suggestion
  const carsPosition = this.model.getCarsPosition();
  return String.raw`<div class="mt-4 d-flex">
    ${carsPosition.map((carPosition) => this.#getCarTemplate(carPosition)).join('')}
```

꼭 이럴 필요는 없지만 단수와 복수에 대한 표현이 명확한게 어떨까요?

Reply:
[34a50ec](https://github.com/next-step/js-racingcar/pull/124/commits/34a50ec73e455b09fc5cd316d6f67a8703490d5d) 여기서 변경해주었어요!